CyberHealth/assessment/admin.py

```diff
@@ -1,6 +1,5 @@
 from django.contrib import admin
-from assessment.models import Organisation, OrganisationRegion, OrganisationType, Answer, Question, Choice, \
-    PathwayGroup, Pathway
+from assessment.models import *
```

Comment:
Could we please do an explicit import - I don't want things being added by accident :)